extensions/sql-migration/src/dashboard/migrationsListTab.ts

```diff
@@ -346,6 +348,13 @@ export class MigrationsListTab extends TabBase<MigrationsListTab> {
 						getMigrationMode(m2),
 						sortDir));
 				return;
+			case TableColumns.type:
+				migrations.sort(
+					(m1, m2) => this.stringCompare(
+						getMigrationType(m1),
+						getMigrationType(m2),
+						sortDir));
+				return;
```

Comment:
I wonder if adding "Type" column here is needed as it does not apply across all migration types.  Schema/data migration only applies to SQL DB and empty for SQL MI/SQL VM.

Reply:
ok. maybe remove it. For SQLMI/VM, it shows "Backup - restore"

Reply:
Confirmed with Abhishek. it is ok to include this column.